api-compact.yaml

```diff
@@ -85,9 +91,11 @@ x-amazon-apigateway-integration: &apigateway_integration
   type: "http"
   requestParameters:
     integration.request.header.Authorization: "method.request.header.Authorization"
+    integration.request.header.Host: "${stageVariables.enterprise_host}"
```

Comment:
My only concern here, and it is a very small concern, is that if we decide to change the api host name at some point in the future from `api.edx.org` to, say, `enterprise-api.edx.org`, we'll need to remember to change this value as well or else we'll have broken integrations.  Like I said, however, this is probably a fringe case -- I assume we will continue to utilize `api.edx.org` as our single logical integration host for at least another year, if not for several.